public/src/components/Groups.tsx

```diff
@@ -33,19 +35,15 @@ const Groups = (): JSX.Element => {
         return true
     }
 
-    const updateGroupStatus = (group: Group, status: Group_GroupStatus) => {
-        actions.updateGroupStatus({ group, status })
-    }
-
     const GroupButtons = ({ group }: { group: Group }) => {
         const buttons: JSX.Element[] = []
         if (isPendingGroup(group)) {
-            buttons.push(<span onClick={() => updateGroupStatus(group, Group_GroupStatus.APPROVED)} className="badge badge-primary clickable">Approve</span>)
+            buttons.push(<DynamicButton onClick={() => actions.updateGroupStatus({ group, status: Group_GroupStatus.APPROVED })} text={"Approve"} color={Color.BLUE} type={ButtonType.BADGE} />)
```

Comment:
These `button.push` lines are a bit long. Maybe split onto multiple lines somehow. Or optionally place the `text` field/label earlier on the line to make it easier to identify the "button" it belongs to.